### PR TITLE
Test GitHub

### PR DESCRIPTION
**Pull request by dairoquintero**

### CODE REVIEW

File: .github/workflows/gradle.yml [hunk 1]
```diff
@@ -10,18 +10,21 @@ on:
...
+      - name: Set up JDK 11
+        uses: actions/setup-java@v2
+        with:
+          java-version: '11'
+          distribution: 'adopt'
+      - name: Build with Gradle
+        env:
+          ACCESS_TOKEN: ${{ secrets.ACCESS_TOKEN }}
+        run: | 
+              ./gradlew build
```
Comment (aperdomob) [inline .github/workflows/gradle.yml]: why build need the secret, the stage should be called "Test"

File: src/test/java/co/com/api/test/ApiHttpBinTest.java [hunk 2]
```diff
@@ -1,5 +1,9 @@
 package co.com.api.test;
```
Comment (aperdomob) [inline src/test/java/co/com/api/test/ApiHttpBinTest.java]: why these changes, are you updated with main?

**Review by aperdomob — CHANGES_REQUESTED** [review]

File: .github/workflows/gradle.yml (hunk 1 above)
Comment (dairoquintero) [reply .github/workflows/gradle.yml]: Actually when the pipeline ./gradlew build. Execute: checkStyleTest and test include ApiHttpBinTest and ApiGithubTest

File: src/test/java/co/com/api/test/ApiHttpBinTest.java (hunk 2 above)
Comment (dairoquintero) [reply src/test/java/co/com/api/test/ApiHttpBinTest.java]: Because when i configurate the spaces and other checkstyle configuration, with the indention in intellij the spaces are removed and i need to restores this.

**Review by dairoquintero — COMMENTED** [review]
Done

File: .github/workflows/gradle.yml (hunk 1 above)
Comment (dairoquintero) [reply .github/workflows/gradle.yml]: Seperate build and test in the last commit

**Review by dairoquintero — COMMENTED** [review]
Ok

**Review by aperdomob — APPROVED** [review]
